### ISSUE TITLE
Publish on F-Droid

### ISSUE THREAD

**Issue by konradmoesch** [Clon1998/mobileraker#394]
### Problem Description
Since I don't use Google Play (and don't want to), I would like to have Mobileraker published on [F-Droid](https://f-droid.org/)
This would enable (auto-)updates for users without the google play store

### Proposed Solution
Mobileraker would be added to the F-Droid repository, making it available via the F-Droid client apps.
To make an app available in the official repo, one creates a merge request (if the original app author does not oppose the inclusion, of course) on the [Fdroiddata gitlab repository](https://gitlab.com/fdroid/fdroiddata/-/merge_requests). Any issues (e.g. packaging, unclear licenses, maybe enabling reproducible builds, etc.) would be checked against and solved in this MR.
An even easier option would be to include the app in the [IzzyOnDroid F-Droid repository](https://apt.izzysoft.de/fdroid/index/info), which is a large, unofficial f-droid repo with less strict inclusion policies.

### Alternatives Considered
Alternatives would be using google play (if available) or manually building/downloading the apk

### Additional Context
Having worked with F-Droid and its repo before, I am of course willing to help:)

---

**Checklist**
To help us understand your feature request, please ensure you've covered the following points:
- [x] Provided a clear description of the problem you're facing.
- [x] Clearly outlined the proposed solution or feature.
- [x] Mentioned any alternative solutions or ideas you've considered.
- [x] Included any relevant context or supporting materials.

Your input is valuable in shaping the development of our project. Thank you for taking the time to submit your feature request!

**Comment by Clon1998** [comment]
There were several discussion about this already.
Tl;Dr; Due to my licensing and used packages, there currently is no way to publish it to F-Droid.

If you still want to try it you can open a PR on the respective repos.
